opds.py

```diff
@@ -578,163 +578,168 @@ def detailed_author(cls, contributor):
 class AcquisitionFeed(OPDSFeed):
 
     FACET_REL = "http://opds-spec.org/facet"
-    NO_CACHE = object()
+    # TODO: Is this still used?
     FEED_CACHE_TIME = int(Configuration.get('default_feed_cache_time', 600))
 
     @classmethod
-    def groups(cls, _db, title, url, lane, annotator,
-               cache_type=None, force_refresh=False, facets=None,
-               search_engine=None, search_debug=False
+    def groups(cls, _db, title, url, worklist, annotator,
+               facets=None, max_age=None, 
+               search_engine=None, search_debug=False,
     ):
         """The acquisition feed for 'featured' items from a given lane's
         sublanes, organized into per-lane groups.
 
         :param facets: A GroupsFacet object.
 
-        :return: CachedFeed (if use_cache is True) or unicode
+        :return: A Unicode string containing a (potentially cached) OPDS feed.
         """
-        works_and_lanes = None
-        if lane.children:
-            # Since this lane has children, it's reasonable to at least
-            # try to create a groups feed for it.
```

Comment:
I'm pretty sure it's reasonable to do this whether or not the lane has children, but I'm not sure what happens when the lane doesn't have children.